docs/use-cases-header.md

```diff
@@ -1,3 +1,7 @@
 # <a name="use-cases"></a>Use cases
 
-ECS fields can be applied to common use cases.
+These are example on how ECS fields can be used in different use cases. Most use
+cases not only contain ECS fields but additional fields which are not in ECS to
+describe the full use case. The fields which are not in ECS are italic.
```

Comment:
@andrewkroh `are italic` sounds strange but not sure how to put it in english.

Reply:
I suggest "are **in** italic".

Reply:
thanks, updated.